pkg/audit/cursor.go

```diff
@@ -49,64 +54,78 @@ func (cursor *Cursor) NextStripe(ctx context.Context) (stripe *Stripe, err error
 	var path storj.Path
 	var more bool
 
-	pointerItems, more, err = cursor.pointerdb.List("", cursor.lastPath, "", true, 0, meta.None)
-	if err != nil {
-		return nil, err
+	var attempts = 0
+	for attempts < maxListAttempts {
+		pointerItems, more, err = cursor.pointerdb.List("", cursor.lastPath, "", true, 0, meta.None)
```

Comment:
Won't this always return the same list of pointers in a specific moment in time? So if `pointerItems` will be empty the first time then there is small chance that something will change?

Reply:
okay I see what you mean-- hopefully my latest change (re-assigning cursor.lastPath) within this loop will help.

Reply:
consider maybe having NextStripe return 'more' instead of all this retrying